Functions/ImageProcessing/xASL_im_ASLSubtractionAveraging.m

```diff
@@ -174,15 +165,6 @@
 
 % We skip the computation of Control4D if it is provided
 if nargin>=5 && ~isempty(Control4D) % same as above
-	if ischar(Control4D) && xASL_exist(Control4D, 'file')
-		% Load NII and update the JSON information from it
-		[Control4D, json] = xASL_io_Nifti2Im(Control4D, [], [], true);
-		
-		x.Q.EchoTime_Control4D = json.Q.EchoTime;
-		x.Q.InitialPLD_Control4D = json.Q.Initial_PLD;
-		x.Q.LabelingDuration_Control4D = json.Q.LabelingDuration;
-	end
-
     if ~isnumeric(Control4D)
```

Comment:
I would sort the nargins numerically

Reply:
Sure - that was you who wrote that...

Fixed. You can resolve